### PR TITLE
events: Add skeleton slides for M4 event (conference)

### PR DESCRIPTION
**Pull request by razvand**
Add reveal-md + markdown-pp skeleton contents, similar to slides for other events. There are 5 sections with owners for each, marked in each `.md` file in the `.slides/` directory:

* overview: @razvand
* methodology: @RazvanN7
* infrastructure: @Sergiu121 (+ @VladNastase + @jokeswar)
* repositories: @teodutu, @eggertkarl, @gabrielmocanu
* conclusion: @razvand

### CODE REVIEW

File: website/docs/events/m4-slides/.slides/conclusion.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+### Results
+
+* methodology
+* infrastructure
+* repositories
+
+----
+
+### What Can I Do?
```
Comment (teodutu) [inline website/docs/events/m4-slides/.slides/conclusion.md:17]: :fist_raised: :eggplant:

File: website/docs/events/m4-slides/.slides/conclusion.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+* join Discord
```
Comment (teodutu) [inline website/docs/events/m4-slides/.slides/conclusion.md]: Add the link here: www.bit.ly/OpenEduHub

File: website/docs/events/m4-slides/.slides/conclusion.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+* take part in discussions, provide feedback, provide suggestions, provide use cases
+* use current and future content & infrastructure
+* contribute to content repositories: review, add new content
+* contribute to infrastructure repositories
+* think of porting your own content
```
Comment (teodutu) [inline website/docs/events/m4-slides/.slides/conclusion.md]:
```suggestion
* think of porting your own content to the OpenEdu methodology
```

**Review by teodutu — CHANGES_REQUESTED** [review]
I think it's fine, the sections make sense and they cover most of our content. I only made a few suggestions.

**Review by RazvanN7 — APPROVED** [review]

**Review by teodutu — APPROVED** [review]
:+1:

**Pull request merged**